docs/language-server/protocol-language-server.md

```diff
@@ -717,16 +724,17 @@ may be expanded in future.
  * @param creationTime creation time
  * @param lastAccessTime last access time
  * @param lastModifiedTime last modified time
- * @param kind type of [[FileSystemObject]], can be:
- * `Directory`, `File`, `Other`
+ * @param kind type of [[FileSystemObject]], can be: `Directory`, `File`, `Other`
  * @param byteSize size in bytes
+ * @param checksum: The SHA3-224 checksum of the file or directory contents.
```

Comment:
By `directory contents` you mean the checksum of file names, or checksum of their contents?

And anyway, I'm not sure if I would put the checksum here: I expect getting attributes is a very light operation, but counting the checksum sometimes is not.

Reply:
This is a good point. Does the `file/checksumBytes` operation suffice or would you want some other mechanism for it?

Reply:
I've added a separate `file/checksum` as suggested by Dmitry below.